web/components/application/hacker/pages/PageTwo/PageTwo.js

```diff
@@ -17,33 +20,161 @@ class PageTwo extends Page {
+            }}
+          />
+        </CheckboxGroup>
+
+        <TextInput
+          label="Github/BitBucket/Gitlab link (optional)"
+          name="github-link"
+          className="margin-ends-giga"
+          placeholder="https://github.com/JohnDoe"
+          value={githubLink}
+          onChange={newGithubLink => this.updateApplication({ githubLink: newGithubLink })}
+          onBlur={() => this.setFieldAsBlurred('githubLink')}
+          error={this.getErrorIfBlurred('githubLink')}
+        />
+        <TextInput
+          label="Personal website/portfolio link (optional)"
+          name="personal-website-link"
+          className="margin-ends-giga"
+          placeholder="https://johndoe.com/"
+          value={personalWebsiteLink}
+          onChange={newPersonalWebsiteLink => this.updateApplication({
+            personalWebsiteLink: newPersonalWebsiteLink })}
+          onBlur={() => this.setFieldAsBlurred('personalWebsiteLink')}
+          error={this.getErrorIfBlurred('personalWebsiteLink')}
+        />
+        <TextInput
+          label="LinkedIn (optional)"
+          name="linkedin-link"
+          className="margin-ends-giga"
+          placeholder="https://www.linkedin.com/in/johndoe/"
+          value={linkedInLink}
+          onChange={newLinkedInLink => this.updateApplication({ linkedInLink: newLinkedInLink })}
+          onBlur={() => this.setFieldAsBlurred('linkedInLink')}
+          error={this.getErrorIfBlurred('linkedInLink')}
+        />
+
+        <TextInput
+          label="Resume Link (make sure the link is publicly accessible)"
+          name="resume-link"
+          className="margin-ends-giga"
+          placeholder="https://drive.google.com/open?id=0B0GZwoww_XiFMHJENWlYN0R2RlE"
+          value={resumeLink}
+          onChange={newResumeLink => this.updateApplication({ resumeLink: newResumeLink })}
+          onBlur={() => this.setFieldAsBlurred('resumeLink')}
+          error={this.getErrorIfBlurred('resumeLink')}
+        />
+        <TextArea
+          label={(<div><img className="vertical-align-top emoji" alt="🏗️" src={crane} /> What are you interested in building at nwHacks? Tell us about an idea you have, and why it gets you excited.</div>)}
+          name="longanswer-1"
+          className="margin-ends-giga"
+          placeholder="Max 750 characters"
+          value={interestForNwHacks}
+          maxLength={750}
+          rows={7}
+          onChange={newInterestForNwHacks => this.updateApplication({
+            interestForNwHacks: newInterestForNwHacks })}
+          onBlur={() => this.setFieldAsBlurred('interestForNwHacks')}
+          error={this.getErrorIfBlurred('interestForNwHacks')}
+        />
+        <TextArea
+          label={(<div><img className="vertical-align-top emoji" alt="🏅" src={medal} /> Tell us about a recent project you&apos;ve worked on that you&apos;re proud of! (It doesn&apos;t have to be technical)</div>)}
```

Comment:
for future reference, you can also do
`{Tell us about a recent project you've worked on that you're proud of! (It doesn't have to be technical)}`

Reply:
then you don't need to replace all the ' with &apos;